std/algorithm/iteration.d

```diff
@@ -4835,13 +4837,64 @@ if (isInputRange!Range && is(typeof(binaryFun!pred(r.front, r.front)) == bool))
     assert(equal(uniq([ 1, 1, 2, 1, 1, 3, 1]), [1, 2, 1, 3, 1]));
 }
 
+@safe unittest
```

Comment:
how about adding the magic `///` to make this a public unittest and thus appear on the docs?